tests/contrib/logging/test_tracer_logging.py

```diff
@@ -202,9 +202,9 @@ def test_logs_with_basicConfig(run_python_code_in_subprocess, ddtrace_run_python
 
         code = """
 import logging
-import ddtrace
```

Comment:
Why were we able to remove this? Can you help me understand why removing this line doesn't result in line 208/210 messing up below?
```python
ddtrace_logger = logging.getLogger('ddtrace')
```

Reply:
That was an oversight on my part. Thanks for catching it.